timewatch/timewatch.py

```diff
@@ -1,6 +1,7 @@
 import requests
 import BeautifulSoup
 
+from tqdm import *
```

Comment:
Could you only import what you need?
`from ... import *` is considered bad practice